slurm-script/kb_init.sh

```diff
@@ -9,7 +9,9 @@
 set -e # Abort if any command fails
 
 export JAVA_OPTS="-Xmx80G"
-source ./bin/python-virtual-environments/env/bin/activate
-pip install -r ./resources/requirements.txt
+
+source ~/softwares/python-virtual-environments/env3/bin/activate
```

Comment:
This is different from the virtual-environment in the PATH in `env.sh`. Is that meant to be?

Reply:
Nevermind, my confusion.